src/components/WalletAssetsHeadline.vue

```diff
@@ -61,6 +61,15 @@ export default class WalletAssetsHeadline extends Mixins(TranslationMixin, Loadi
   verifiedOnlySwitch = false;
   zeroBalanceSwitch = false;
 
+  get computedClasses(): string {
+    const baseClass = ['wallet-assets-headline__content'];
+    console.log('this.assetsFiatAmount', this.assetsFiatAmount);
```

Comment:
Please don't forget to remove this console.log()

Reply:
thx 😊